kolena/io.py

```diff
@@ -52,7 +52,9 @@ def _deserialize_dataobject(x: Any) -> Any:
 
 
 def _serialize_dataobject_str(x: Any) -> Any:
-    return json.dumps(x, cls=DataObjectJSONEncoder)
+    if isinstance(x, list) or isinstance(x, dict) or isinstance(x, DataObject):
```

Comment:
nit: can be `isinstance(x, (list, dict, DataObject))`.

Reply:
Thanks!